packages/ag-charts-enterprise/src/features/annotations/annotations.ts

```diff
@@ -568,7 +568,9 @@ export class Annotations extends _ModuleSupport.BaseModuleInstance implements _M
         const anchor = Vec2.add(rect, Vec2.from(0, rect.height + 4));
         const fallback = { y: rect.y - 4 };
         this.colorPicker.setAnchor(anchor, fallback);
-        this.textSizePopover.setAnchor(anchor);
+
+        const textAnchor = { x: rect.x + 34 - 1, y: rect.y + rect.height - 1 };
```

Comment:
This `34` seems to be shared with the `toobar.css`, is there a way we can ensure the consistency of this JS code vs. the CSS definition?

Reply:
It's a hack for sure. There's not really a good sharing mechanism between CSS and JS that'll work for what we want. It not the only issue too - if the toolbar items change order, this won't reflect that either. I think it will be worth revisiting this once we've implemented more UI throughout charts and see if we can tidy up this implementation, or we find a better design that doesn't have these issues